pkgs/fc/ceph/src/fc/ceph/lvm.py

```diff
@@ -167,20 +261,18 @@ def _create(self, disk: str, size: str, vg_name: str):
         )
 
     @staticmethod
-    def ensure_vg(vg_name: str, disk: str):
+    def ensure_vg(vg_name: str, base_device: Optional[GenericBlockDevice]):
```

Comment:
1. Is this indeed called somewhere without a base_device?
2. If so, then this is not really an `ensure` anymore because it cannot create the vg if needed.